react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -70,6 +73,26 @@ const Dashboard: React.FC = () => {
     }
   };
 
+  const handleActionSubmit = async () => {
```

Comment:
naming not good. It should be `handleDeleteCustomer`

Reply:
i done